.github/workflows/ci-install-script.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
                 docker run -t -v $PWD/install.sh:/install.sh:ro \
                     -e GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} \
                     ${{ matrix.container-image }} \
-                        bash -c 'apt-get update && apt-get install -y curl && /install.sh && calyptia version'
+                        bash -c 'apt-get update && apt-get install -y curl && cli_VERSION=v1.19.1 /install.sh && calyptia version'
```

Comment:
Should we just remove the 1.19.2 release?

Reply:
let's do it i'll remove this argument from here

Reply:
done